functest/package_test.go

```diff
@@ -11,23 +11,35 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package functest
 
 import (
+	"bytes"
 	"path/filepath"
 	"testing"
 
+	cmd "github.com/appsody/appsody/cmd"
 	"github.com/appsody/appsody/cmd/cmdtest"
 )
 
 func TestPackage(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	args := []string{"stack", "package", filepath.Join("..", "cmd", "testdata", "starter")}
-	_, err := cmdtest.RunAppsody(sandbox, args...)
+	var outBuffer bytes.Buffer
+	log := &cmd.LoggingConfig{}
+	log.InitLogging(&outBuffer, &outBuffer)
+
+	stackDir := filepath.Join("..", "cmd", "testdata", "starter")
```

Comment:
Use `TestDirPath` here

Reply:
Of course, yes! Done, and changed for most places throughout the code!